swo_aws_extension/flows/steps/setup_parameters.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from copy import deepcopy
+
+from mpt_extension_sdk.flows.pipeline import Step
+from mpt_extension_sdk.mpt_http.base import MPTClient
+from mpt_extension_sdk.mpt_http.mpt import update_order
+
+from swo_aws_extension.flows.order import InitialAWSContext
+
+logger = logging.getLogger(__name__)
+
+
+class SetParametersVisibleStep(Step):
+    def set_hidden_paramter(self, parameter, hidden=False):
+        """
+        Update the visibility of the parameter to read-only.
+
+        Args:
+            parameter (dict): The parameter to update.
+        """
+        new_parameter = deepcopy(parameter)
+        if "constraints" not in new_parameter:
+            new_parameter["constraints"] = {}
+        new_parameter["constraints"]["hidden"] = hidden
+        return new_parameter
+
+    def process_parameters(self, parameters, always_shown=None, always_hidden=None):
```

Comment:
Could you point me where do you use two function parameters `always_shown` and `always_hidden` in real scenarious? I see that you have it only for tests, probably for now there is no need for such kind of function parameters?